### PR TITLE
[ch3212] Some wording change about organizational chart PR

### PR DESCRIPTION
**Pull request by cacahouwete**
https://app.clubhouse.io/enmarche/story/3212/cr%C3%A9er-dans-l-espace-r%C3%A9f%C3%A9rent-une-nouvelle-tab-%C3%A9quipe-d%C3%A9partementale-qui-pr%C3%A9sentera-l-organigramme

### CODE REVIEW

File: src/Admin/OrganizationalChartItemAdmin.php [hunk 1]
```diff
@@ -48,8 +52,12 @@ protected function configureShowFields(ShowMapper $show)
     protected function configureFormFields(FormMapper $form)
     {
         $form
-            ->add('label')
-            ->add('parent', ModelListType::class)
+            ->add('label', null, [
+                'label' => 'Fonction'
+            ])
+            ->add('parent', ModelListType::class, [
+//                'btn_list' => 'toto',
```
Comment (thomasbisignani) [inline src/Admin/OrganizationalChartItemAdmin.php]: why this comment?

Comment (Remg) [inline src/Admin/OrganizationalChartItemAdmin.php]: please remove this

File: features/referent-organizational-chart.feature [hunk 2]
```diff
@@ -37,7 +37,7 @@ Feature: Make sure we can see and interact with organizational chart
       | Prénom           | Dupoint                  |
       | Mail             | test@test.fr             |
       | Téléphone        | 0612345678               |
-      | Addresse postale | 1 avenue des chez élisée |
+      | Adresse postale | 1 avenue des chez élisée |
```
Comment (ottaviano) [inline features/referent-organizational-chart.feature]: with one space for align `|` it will be perfect :)

File: translations/messages.fr.yml [hunk 3]
```diff
@@ -10,6 +10,8 @@ common.show_less: Voir moins
 common.sector: Secteur
 pending: En attente
 approved: Approuvé
+group: Groupe
```
Comment (ottaviano) [inline translations/messages.fr.yml]: where is it used?

Comment (cacahouwete) [reply translations/messages.fr.yml]:
```xml

            <call method="setSubClasses">
                <argument type="collection">
                    <argument key="person">AppBundle\Entity\ReferentOrganizationalChart\PersonOrganizationalChartItem</argument>
                    <argument key="group">AppBundle\Entity\ReferentOrganizationalChart\GroupOrganizationalChartItem</argument>
                </argument>
            </call>
```

The Id is translated

**Pull request merged**